backend/src/graphql/resolver/ContributionMessageResolver.ts

```diff
@@ -57,6 +62,11 @@ export class ContributionMessageResolver {
         await queryRunner.manager.update(DbContribution, { id: contributionId }, contribution)
       }
       await queryRunner.commitTransaction()
+      await EVENT_CONTRIBUTION_MESSAGE_CREATE(
+        user,
+        { id: contributionMessage.contributionId } as DbContribution,
```

Comment:
Like at the bottom it feels strange to define it like this. At line 62 it seem to use the same object.